api/v1alpha1/packagebundlecontroller_webhook.go

```diff
@@ -0,0 +1,104 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
+)
+
+type activeBundleValidator struct {
+	Client  client.Client
+	decoder *admission.Decoder
+}
+
+func InitActiveBundleValidator(mgr ctrl.Manager) error {
+	mgr.GetWebhookServer().
+		Register("/validate-packages-eks-amazonaws-com-v1alpha1-packagebundlecontroller",
+			&webhook.Admission{Handler: &activeBundleValidator{
+				Client: mgr.GetClient(),
+			}})
+	return nil
+}
+
+func (v *activeBundleValidator) Handle(ctx context.Context,
+	req admission.Request) admission.Response {
+
+	pbc := &PackageBundleController{}
+	err := v.decoder.Decode(req, pbc)
+	if err != nil {
+		return admission.Errored(http.StatusInternalServerError,
```

Comment:
Is this just the first time I noticed the tab stop is 8 here?

Reply:
I have no idea. My editor displays each indentation as four spaces regardless of what it really is. I thought gofmt took care of the rest?

I can look into editor settings if you think I have something misconfigured.